setup.py

```diff
@@ -19,113 +19,127 @@
 from distutils import dep_util
 from distutils import log
 
+import pkgconfig
```

Comment:
maybe this `import pkgconfig` should be moved down closer to where it is used, after `if USE_SYSTEM_BROTLI:`

this way it will not be required unless one explicitly request to build against system brotli